### PR TITLE
[jiminy_py] Add jupyter notebook tutorial to document basic usage.

### PR DESCRIPTION
**Pull request by matthieuvigne**
Related to #34 

Add a jupyter notebook to show an interactive example of using jiminy and the BasicSimulator class.

Add function in viewer to directly display results of a logfile (without having to manually do the conversion to a state).

### CODE REVIEW

**Comment by matthieuvigne** [comment]
@duburcqa There are many other things we could add to this tutorial, but this is already a start.

Ideas:

 - Adding constants / new variables to a log  - doesn't make much sense on a simple pendulum, but would on a more complex system.
 - A multi-robot simulation with interesting coupling ?
 - Contacts ?
 - Adding sensors, changing the simulation configuration.
 - Adding flexibilities ?
 - Illustration of contact constraint: I'm waiting on having a working simulation of a wheel turning -  a segway would be a good example.

File: python/jiminy_py/src/jiminy_py/viewer.py [hunk 1]
```diff
@@ -32,7 +33,8 @@ class Viewer:
 
     def __init__(self, robot, use_theoretical_model=False,
                  urdf_rgba=None, robot_index=0,
-                 backend=None, window_name='python-pinocchio', scene_name='world'):
+                 backend=None, window_name='python-pinocchio', scene_name='world',
+                 urdf_root_path = None):
```
Comment (duburcqa) [inline python/jiminy_py/src/jiminy_py/viewer.py:37]: It think 'mesh_root_path' would be more appropriate.

Comment (duburcqa) [reply python/jiminy_py/src/jiminy_py/viewer.py:37]: Besides,  'mesh_root_path' should come right after 'use_theoretical_model' IMHO.

File: python/jiminy_py/src/jiminy_py/viewer.py [hunk 2]
```diff
@@ -432,8 +435,8 @@ def display(self, evolution_robot, speed_ratio, xyz_offset=None):
 
 
 def play_trajectories(trajectory_data, xyz_offset=None, urdf_rgba=None, speed_ratio=1.0,
-                      backend='gepetto-gui', window_name='python-pinocchio', scene_name='world',
-                      close_backend=None):
+                      backend=None, window_name='python-pinocchio', scene_name='world',
+                      close_backend=None, urdf_root_path = None):
```
Comment (duburcqa) [inline python/jiminy_py/src/jiminy_py/viewer.py:439]: same

File: python/jiminy_py/src/jiminy_py/viewer.py [hunk 3]
```diff
@@ -465,7 +471,8 @@ def play_trajectories(trajectory_data, xyz_offset=None, urdf_rgba=None, speed_ra
         use_theoretical_model = trajectory_data[i]['use_theoretical_model']
         robot = Viewer(robot, use_theoretical_model=use_theoretical_model,
                        urdf_rgba=urdf_rgba[i] if urdf_rgba is not None else None, robot_index=i,
-                       backend=backend, window_name=window_name, scene_name=scene_name)
+                       backend=backend, window_name=window_name, scene_name=scene_name,
+                       urdf_root_path = urdf_root_path)
```
Comment (duburcqa) [inline python/jiminy_py/src/jiminy_py/viewer.py:475]: same.

File: python/jiminy_py/src/jiminy_py/viewer.py [hunk 4]
```diff
@@ -432,8 +435,8 @@ def display(self, evolution_robot, speed_ratio, xyz_offset=None):
 
 
 def play_trajectories(trajectory_data, xyz_offset=None, urdf_rgba=None, speed_ratio=1.0,
-                      backend='gepetto-gui', window_name='python-pinocchio', scene_name='world',
-                      close_backend=None):
+                      backend=None, window_name='python-pinocchio', scene_name='world',
+                      close_backend=None, urdf_root_path = None):
     """!
     @brief      Display robot evolution in Gepetto-viewer at stable speed.
```
Comment (duburcqa) [inline python/jiminy_py/src/jiminy_py/viewer.py:441]: in the chosen viewer

File: python/jiminy_py/src/jiminy_py/viewer.py [hunk 5]
```diff
@@ -493,3 +500,58 @@ def play_trajectories(trajectory_data, xyz_offset=None, urdf_rgba=None, speed_ra
...
+
+        # Create state sequence
+        evolution_robot = []
+        for i in range(len(t)):
+            evolution_robot.append(State(qe[i].T, None, None, t[i]))
+
+        traj = {'evolution_robot': evolution_robot,
+                'robot': robot,
+                'use_theoretical_model': use_theoretical_model}
+        trajectories.append(traj)
```
Comment (duburcqa) [inline python/jiminy_py/src/jiminy_py/viewer.py:555]: skip line

**Pull request merged**